src/http/kv/create_namespace.rs

```diff
@@ -0,0 +1,32 @@
+use cloudflare::apiclient::APIClient;
+use cloudflare::response::APIFailure;
+use cloudflare::workerskv;
+
+use crate::settings::global_user::GlobalUser;
+use crate::settings::project::Project;
+
+use super::{api_errors_to_message, new_api_client, KvNamespace};
+
+pub fn create_namespace(
+    user: GlobalUser,
+    project: &Project,
+    title: String,
+) -> Result<KvNamespace, failure::Error> {
+    let client = new_api_client(user);
+
+    let res = client
+        .request(&workerskv::CreateNamespace {
+            account_identifier: &project.account_id,
+            params: workerskv::CreateNamespaceParams { title },
+        })
+        .unwrap_or_else(|failure| match failure {
+            APIFailure::Error(_, errors) => {
+                panic!("{}", api_errors_to_message(errors));;
```

Comment:
panics should be replaced by our own errors, these are for illustration purposes only.